### PR TITLE
Add step to verify SignalR server with custom CA.

### PR DESCRIPTION
**Pull request by gokhansengun**
Attempt to solve #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,5 +105,12 @@ There is a [sample application](https://github.com/TargetProcess/signalr-client-
 
 ## Troubleshooting
 
+#### Using a SignalR server running HTTPS with custom a certificate chain
+Add `WEBSOCKET_CLIENT_CA_BUNDLE` variable to the environment. This variable should point to the CA (Certificate Authority) bundle including the certificate to verify the server's certificate. Below is an example on a Ubuntu based system.
```
Comment (coin8086) [inline README.md]: So what about Windows?

**Comment by gokhansengun** [comment]
Nobody's concern except me.

**Comment by coin8086** [comment]
@gokhansengun I'd suggest you to wait a little longer, or mention someone explicitly.

File: README.md [hunk 2]
```diff
@@ -105,5 +105,12 @@ There is a [sample application](https://github.com/TargetProcess/signalr-client-
...
+
+```
+export WEBSOCKET_CLIENT_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
```
Comment (coin8086) [inline README.md]: What about Python on Windows?

**Pull request closed without merging**